### ISSUE TITLE
Files missing in released gemfile

### ISSUE THREAD

**Issue by oliverklee** [braingourmets/currency_select#136]
_Originally posted by @silent-e in https://github.com/braingourmets/currency_select/issues/135#issuecomment-1656342794_
            
> Hey Oliver.  I just pulled this new version to check it against the app I used to make my recent PR and I think something happened.  I'm missing in the lib directory…

```
currency_select_tag.rb
form_builder.rb
form_options_helper.rb
to_currency_select_tag.rb
```

> Looks like your gemspec file doesn't include them.  It only has `lib/currency_select.rb`.

**Comment by oliverklee** [comment]
I have fixed this and just released version 5.0.1. Could you please recheck if this works for you now, @silent-e? Thanks! :pray:

**Comment by silent-e** [comment]
That did it.  My tests are running.  Thanks for the quick fix.
